src/utils/particle/drawer.ts

```diff
@@ -91,7 +93,11 @@ export class Drawer {
                 ctx.rect(particle.position.x - radius, particle.position.y - radius, radius * 2, radius * 2);
                 break;
             case ShapeType.triangle:
-                Drawer.subDrawShape(ctx, particle.position.x - radius, particle.position.y + radius / 1.66, radius * 2, 3, 2);
+                const startX = particle.position.x - radius;
+                const startY = particle.position.y + radius / 1.66;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 1.66](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)